src/sentry/conf/server.py

```diff
@@ -1660,9 +1661,7 @@ def create_partitioned_queues(name):
             "REDIS_PORT": "6379",
             "REDIS_DB": "1",
         },
-        "only_if": lambda settings, options: (
-            "snuba" in settings.SENTRY_EVENTSTREAM or "kafka" in settings.SENTRY_EVENTSTREAM
-        ),
```

Comment:
Is this intentional?

Reply:
Yes, the only eventstream backends that exist now are Snuba and Kafka so this is already always True. Essentially all configurations of Sentry have required Snuba for some time.